VirtoCommerce.Storefront.Model/Customer/Contracts/MemberDto.cs

```diff
@@ -0,0 +1,17 @@
+using System.Collections.Generic;
+
+namespace VirtoCommerce.Storefront.Model.Customer.Contracts
```

Comment:
You don't need such separated Dto types, you can deserialize directly to Storefront.Customer domain types

Reply:
The Dtos are needed, as the models are not identical and migth change. There is issue returning Customer.Organization at the moment